sirius-valley/lerni-mobile
# [LER-9] Pill Header

To test this componente you should: 

1) insert the <PillHeader/> component on a screen
2) check the styles matches the figma
